bin/generate-page

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env php
+<?php declare(strict_types=1);
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+require __DIR__ . '/../vendor/codeigniter4/codeigniter4/system/Test/bootstrap.php';
+
+use CodeIgniter\CLI\CLI;
+use Translations\Tests\AbstractTranslationTestCase;
+
+$locales = AbstractTranslationTestCase::$locales;
+
+if ($locales === []) {
+    CLI::error('No locales list available.');
+
+    exit(1);
+}
+
+helper('filesystem');
+
+$enDir = VENDORPATH . 'codeigniter4/codeigniter4/system/Language/en';
+
+if (! is_dir($enDir)) {
+    CLI::error('No "Language/en" directory. Please run "composer update".');
+
+    exit(1);
+}
+
+// get default "en" language for reference
+$enFiles = get_filenames(
+    $enDir,
+    true,
+    false,
+    false
+);
+
+$enItems = [];
+
+foreach ($enFiles as $enFile) {
+    $enFileLines = file($enFile);
+    $enName      = basename($enFile);
+    $pattern     = '/(.*)\'([a-zA-Z0-9_]+?)\'(\s*=>\s*)([\'"].+[\'"]),/u';
+
+    $enItems[$enName] = ['lines' => [], 'count' => 0];
+
+    foreach ($enFileLines as $line) {
+        if (preg_match($pattern, $line, $matches)) {
+            $enItems[$enName]['lines'][$matches[2]] = $matches[4];
+            $enItems[$enName]['count']++;
+        }
+    }
+}
+
+// create build directory
+$buildDir = __DIR__ . '/../page/build';
+
+if (! is_dir($buildDir) && ! mkdir($buildDir)) {
```

Comment:
Will **page/** always exist? Making this recursive could improve its success rate if there is a question. 
```suggestion
if (! is_dir($buildDir) && ! mkdir($buildDir, 0777, true)) {
```

Reply:
Never mind, I see below **page** is added 👌